src/resources/postcss/tickets-v1.pcss

```
@@ -445,7 +445,8 @@
 	}
 
 	&.tribe-tickets,
-	&.event-tickets {
+	&.event-tickets,
+	& .tribe-ticket {
```

Was this meant with a space?

```suggestion
	&.tribe-ticket {
```

---

ouch! it's just me moving fast and adding an unnecessary ampersand. The space is ok, but I just followed the two above and used the ampersand when not necessary. Fixed it.